Selenium/YandexLogin/Pages/BasePage.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+
+namespace YandexLogin.Pages
+{
+    public class BasePage
+    {
+        private IWebDriver driver;
```

Comment:
Start fields and properties from the capital letter. Apply it to all your classes within solution.

Reply:
Fixed